_includes/guide.njk

```diff
@@ -50,35 +52,25 @@ layout: layout
   </div>
 </main>
 
-{# <div class="cta-next-guide">
+<div class="cta-next-guide">
   <div class="cta-next-guide__wrapper">
-    {% if next %}
-    <div class="cta-next-guide__heading">
-      Next Guide
-    </div>
-    <a class="cta-next-guide__link gc-analytics-event"
+    {% set nextUrl = macros.nextGuide(page, paths).trim() %}
+    {% if nextUrl !== '' %}
```

Comment:
I noticed when I got to the end of the guides it still showed a next guide but it was pointing at the path landing page.

![image](https://user-images.githubusercontent.com/1066253/53134012-265c6a00-352a-11e9-9030-2462afc1a36b.png)

Reply:
In the current version on web.dev "Accessible to all" is both a) repeated twice b). Links to the same destination as "Return to all guides". Maybe it would be best to not have it in the last guide in a series at all, and leave only "return to all guides" link?